pkg/controller/jobs/job/job_controller_test.go

```diff
@@ -2558,7 +2558,7 @@ func TestReconciler(t *testing.T) {
 				Suspend(true).
 				Obj(),
 			wantJob: *baseJobWrapper.Clone().Toleration(corev1.Toleration{
-				Key:      "tolerationkey2",
+				Key:      "tolerationkey1",
```

Comment:
why?

Reply:
Because now the job stops and pod set info restores.

https://github.com/kubernetes-sigs/kueue/blob/dfa182907281dea39fd227aea58abf1374727bef/pkg/controller/jobs/job/job_controller.go#L183-L189